Source/OrientationLib/IO/AngleFileLoader.h

```diff
@@ -88,8 +88,11 @@ class OrientationLib_EXPORT AngleFileLoader
 
 
   private:
+  public:
```

Comment:
`private: public` should probably be `public:`
Consider deleting one of the empty lines above this.